server/executor/runner.go

```diff
@@ -159,6 +158,13 @@ func (r persistentRunner) processExecQueue(job execReq) {
 		panic(err)
 	}
 
+	if job.test.ServiceUnderTest.Type == model.TriggerTypeTRACEID {
+		traceIDFromParam, err := trace.TraceIDFromHex(job.test.ServiceUnderTest.TRACEID.ID)
+		if err == nil {
```

Comment:
Question: If we get an error here, do we want to continue this test? 

At first glance, the test will not work or give inconsistent results in the error case. If we have another block, like [this](https://github.com/kubeshop/tracetest/pull/1840/files#diff-999879534cbd7435721e05ccc9349cc71a860b82fb1e36172a7e128d2e4d0fa6R34), maybe we could add a comment here signalizing this.